src/kafka/producer.js

```diff
@@ -140,6 +140,7 @@ const
  * rdkafkaConf - specific rdkafka configurations [Refer to configuration doc]{@link https://github.com/edenhill/librdkafka/blob/0.11.1.x/CONFIGURATION.md}
```

Comment:
Can you change this link to point to the master configuration.md instead of the older 0.11.1x